yapapi/runner/ctx.py

```diff
@@ -172,9 +172,19 @@ def begin(self):
         pass
 
     def send_json(self, json_path: str, data: dict):
+        """Sends JSON file to the provider
```

Comment:
does it _send_ the file already? or does it schedule a transfer as one of the steps?